spec/services/find_or_register_user.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
So, did you rename FindOrRegisterUser to RegisterUser? Seems like these two files needs to be deleted